app/src/main/java/com/infomaniak/mail/utils/SentryDebug.kt

```diff
@@ -104,11 +104,43 @@ object SentryDebug {
         )
     }
 
+    fun addAttachmentsBreadcrumb(draft: Draft) = with(draft) {
+
+        var count = 1
+        val data = mutableMapOf<String, Any>()
+
+        fun String.keyPad(): String = padStart(length = 15)
+        fun Int.countPad(): String = toString().padStart(length = 2, '0')
+        fun count(): String = "${count.countPad().also { count++ }}."
+        fun format(index: Int): String = (index + 1).countPad()
+
+        data[count() + "email".keyPad()] = AccountUtils.currentMailboxEmail.toString()
+
+        data[count() + "draft".keyPad() + " - localUuid"] = localUuid
+        data[count() + "draft".keyPad() + " - remoteUuid"] = remoteUuid.toString()
+        data[count() + "draft".keyPad() + " - action"] = action?.name.toString()
+        data[count() + "draft".keyPad() + " - mode"] = when {
```

Comment:
You could create an intermediate variable with `"count() + "draft".keyPad() + " -"`

Reply:
I think it complicates the readability of the code.